template/info_label.txt

```diff
@@ -17,3 +17,5 @@
 13, spinal cord centerline, PAM50_centerline.nii.gz
 14, point-wise spinal level labels, PAM50_spinal_midpoint.nii.gz
 15, spinal cord mask with discrete spinal levels, PAM50_spinal_levels.nii.gz
+16, spinal dorsal nerve rootlets with discrete spinal levels, PAM50_rootlets.nii.gz
+17, spinal dorsal nerve rootlets center of mass with discrete spinal levels, PAM50_rootlets_mid_center.nii.gz
```

Comment:
It looks like this file ended up being called `PAM50_rootlets_midpoints.nii.gz`?

```suggestion
17, spinal dorsal nerve rootlets center of mass with discrete spinal levels, PAM50_rootlets_midpoints.nii.gz
```

Reply:
Agreed! I wanted to be coherent with the spinal levels midpoint file, we could maybe remove the 's' to be even more coherent?
```suggestion
17, spinal dorsal nerve rootlets center of mass with discrete spinal levels, PAM50_rootlets_midpoint.nii.gz
```

Reply:
That makes sense! Rootlets is already plural. :)